lib/katello/engine.rb

```diff
@@ -229,7 +229,9 @@ def find_assets(args = {})
         api_view :list => 'katello/api/v2/subscription_facet/base_with_root', :single => 'katello/api/v2/subscription_facet/show'
       end
 
-      ::SettingsHelper.send :include, Katello::Concerns::SettingsHelperExtensions
+      ::SettingsController.class_eval do
+        helper Katello::Concerns::SettingsHelperExtensions
```

Comment:
Ah nevermind, it's still a concern but it's included by calling `helper`?  I guess that's consistent with some of how the others are done.